src/software/ai/hl/stp/play/penalty_kick_play_test.cpp

```diff
@@ -12,12 +12,53 @@ class PenaltyKickPlayTest : public SimulatedPlayTestFixture
 {
 };
 
-TEST_F(PenaltyKickPlayTest, test_penalty_kick_play)
+TEST_F(PenaltyKickPlayTest, test_penalty_kick_setup)
 {
-    setBallState(BallState(field().penaltyFriendly(), Vector(0, 0)));
+    setBallState(BallState(field().penaltyEnemy(), Vector(0, 0)));
     addFriendlyRobots(TestUtil::createStationaryRobotStatesWithId(
-        {Point(-3, 2.5), Point(-3, 1.5), Point(-3, 0.5), Point(-3, -0.5), Point(-3, -1.5),
-         Point(4.6, -3.1)}));
+        {Point(-2, -2), Point(-3, -1), Point(-3, 0), Point(-3, 1), Point(-3, 2),
+         Point(2, 2.5)}));
+    setFriendlyGoalie(0);
+    addEnemyRobots(
+        TestUtil::createStationaryRobotStatesWithId({field().enemyGoalCenter()}));
+    setEnemyGoalie(0);
+    setAIPlay(TYPENAME(PenaltyKickPlay));
+    setRefereeCommand(RefereeCommand::PREPARE_PENALTY_US, RefereeCommand::NORMAL_START);
+
+    std::vector<ValidationFunction> terminating_validation_functions = {
+        // This will keep the test running for 9.5 seconds to give everything enough
+        // time to settle into position and be observed with the Visualizer
+        // TODO: Implement proper validation
+        // https://github.com/UBC-Thunderbots/Software/issues/1396
+        [](std::shared_ptr<World> world_ptr, ValidationCoroutine::push_type& yield) {
+            while (world_ptr->getMostRecentTimestamp() < Timestamp::fromSeconds(9.5))
+            {
+                yield();
+            }
```

Comment:
we need to replace this with validation functions here

Reply:
added validation